src/vm-repair/azext_vm_repair/_help.py

```diff
@@ -0,0 +1,40 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['vm repair'] = """
+    type: group
+    short-summary: Auto repair commands to fix faulty VMs. Recommended to use all commands with the --verbose flag.
+    long-summary: >
+        VM repair commands will create a new rescue VM to fix the OS disk of the faulty VM. This is called the 'disk-swap' process. The OS disk of the faulty VM will be copied then attached on the rescue VM's data disk. From the rescue VM, one can diagnose and fix the attached data disk.
+"""
+
+helps['vm repair swap-disk'] = """
+    type: command
+    short-summary: Create a new rescue VM, and attach a copy of the OS disk from the target VM to the rescue VM as a data disk. Recommended to use with the --verbose flag.
+    examples:
+        - name: Disk Swap a VM
+          text: >
+            az vm repair swap-disk -g MyResourceGroup -n myVM --verbose
+        - name: Disk Swap a VM and set the rescue VM auths
+          text: >
+            az vm repair swap-disk -g MyResourceGroup -n myVM --rescue-username username --rescue-password password!234 --verbose
+"""
+
+helps['vm repair restore-swap'] = """
```

Comment:
Command name: "vm repair restore-swap" -> "vm repair restore"?

Reply:
Yes this seems better, but I'll try to figure out the naming and teminology with Mahesh today.